core/src/main/java/org/bitcoinj/wallet/DeterministicSeed.java

```diff
@@ -52,39 +55,70 @@ public class DeterministicSeed implements EncryptableItem {
     @Nullable private final List<String> mnemonicCode; // only one of mnemonicCode/encryptedMnemonicCode will be set
     @Nullable private final EncryptedData encryptedMnemonicCode;
     @Nullable private final EncryptedData encryptedSeed;
-    private long creationTimeSeconds;
+    // Creation time of the seed, or null if the seed was deserialized from a version that did not have this field.
+    @Nullable private Instant creationTime = null;
```

Comment:
Can this be `final`?

Reply:
I'm open for trying this out, but in a different PR.